<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DBMPublicTablePanel</class>
 <widget class="QWidget" name="DBMPublicTablePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DBMPublicTablePanel</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="_publicTableTag">
       <property name="text">
        <string>Initialize Public Table</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_btnSelectAll">
       <property name="text">
        <string>SelectAll</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_btnSelectNone">
       <property name="text">
        <string>SelectNone</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="_cbUserTable">
       <property name="text">
        <string>user</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="_cbATTable">
       <property name="text">
        <string>at</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="_cbDeviceTable">
       <property name="text">
        <string>device</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="_cbEMETable">
       <property name="text">
        <string>eme</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="_cbATUtilizationTable">
       <property name="text">
        <string>atUtilization</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="_cbATUtilizationLogTable">
       <property name="text">
        <string>atUtilizationLog</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
